angular-src/src/app/app.module.ts

```diff
@@ -2,10 +2,11 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { HttpClientModule } from '@angular/common/http';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+// import { DragulaModule } from 'ng2-dragula';
```

Comment:
You can probably remove this comment unless you will be digging through the ditches with this module :wink: